planner/planner.go

```diff
@@ -1014,6 +1018,22 @@ func (p *MyPlanner) GetPlaceDetails(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, place)
 }
 
+func (p *MyPlanner) rateLimiter() gin.HandlerFunc {
+	logger := iowrappers.Logger
+	// 100 requests per hour
+	rate, err := limiter.NewRateFromFormatted("100-H")
```

Comment:
100 per hours. ?

Reply:
Can we test this now ?

Reply:
I tested for lower limit. It works pretty well. Currently it is set to 100 per IP per hour. Should be enough for regular usages.